diff --git "a/icu4c/source/common/unicode/platform.h" "b/icu4c/source/common/unicode/platform.h"
index 2bb2f8b318..161c5e0dad 100644
--- "a/icu4c/source/common/unicode/platform.h"
+++ "b/icu4c/source/common/unicode/platform.h"
@@ -21,6 +21,9 @@
 #ifndef _PLATFORM_H
 #define _PLATFORM_H
 
+#define U_IMPORT
+#define U_STATIC_IMPLEMENTATION
+
 #include "unicode/uconfig.h"
 #include "unicode/uvernum.h"
 
