diff --git "a/CMakeLists.txt" "b/CMakeLists.txt"
index 89fc301c..a2faf62a 100644
--- "a/CMakeLists.txt"
+++ "b/CMakeLists.txt"
@@ -452,6 +452,7 @@ if(WEBP_BUILD_IMG2WEBP)
                         imagedec
                         imageioutil
                         webp
+                        liblzma
                         libwebpmux)
   target_include_directories(img2webp PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/src)
   install(TARGETS img2webp RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
@@ -515,7 +516,7 @@ if(WEBP_BUILD_EXTRAS)
 
   # get_disto
   add_executable(get_disto ${GET_DISTO_SRCS})
-  target_link_libraries(get_disto imagedec)
+  target_link_libraries(get_disto imagedec liblzma zstd)
   target_include_directories(get_disto
                              PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}
                                      ${CMAKE_CURRENT_SOURCE_DIR}/src
